Odata-docs/client/using-blazor-wasm-with-odata-client.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Blazor Wasm with OData Client"
+description: "How to use blazor with odata client"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 1/18/2021
+ms.topic: article
+ 
+---
+# How To Use Blazor Web Assembly with OData Client 
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+## Getting Started
+Start by creating a Blazor WebAssembly project.
+
+Open Visual Studio and click on "Create a new project" then select the "Blazor App template" and click "Next". Configure your project by providing the project name and the location path then click "Create". On the "Create Project" Page that appears, specify the target framework that you want to use and select "Blazor WebAssembly App" then click "Create". A blazor web assembly project will be created like below. 
```

Comment:
My suggestion would be to convert this paragraph into an ordered or unordered list so it's easier to follow